lib/__init__.py

```diff
@@ -0,0 +1,15 @@
+from flask import Flask
+from config import Config
+from flask_socketio import SocketIO
+from flask_mongoengine import MongoEngine
+
+
+app = Flask(__name__)
+app.config.from_object(Config)
+app.config['MONGODB_SETTINGS'] = {
+  'host': 'mongodb://localhost/cc-be'
+}
+db = MongoEngine(app)
+socketio = SocketIO(app)
+
```

Comment:
Can we go over what is the proper code to put in _init_ files in general python.  This seems fairly self explanatory in the sense that it is like a import and setting file.  Is there a case to always put this in here or else where.  Can or should anything else go in these files?

Reply:
Any core packages that we want to be available to modules and the app `circle.py`, live in this file. It creates a cohesive package for `lib`. A good comparison would be to look at the `routes.py` file that contains additional packages and modules being imported, but they're specific to that module.